README.md

```diff
@@ -21,7 +21,8 @@ change the commits in the PR.
 To make changes to the CI environment, the changes should be first committed
 to the `master` branch, and deployed to the `staging` environment.  Once they
 are confirmed to be working, the changes should be merged to the `production`
-branch and deployed to the `production` environment.
+branch and deployed to the `production` environment.  For the centos7 based tests, use `linux-system-roles-centos7` for production and
```

Comment:
The previous lines seem to be wrapped but the new one is not.

Reply:
https://github.com/linux-system-roles/test-harness/pull/87/commits/a1f7e36ec876eb927bb6ddcd25ba619b13dc88c0